src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -138,6 +139,12 @@ private URL getSiteURL() throws Exception {
 		debug_log( "Checking base URL: " + baseUrl );
 		return new URL( baseUrl + getConfig().getString( "key" ) );
 	}
+	private URL getSiteURLBackup() throws Exception {
+		//some users were having issues with said custom post url support
+		String baseUrl = getConfig().getString( "url" ) + "/wp-json/wp/v2/";

```

Comment:
This isn't the correct base URL - see here - https://github.com/WooMinecraft/WooMinecraft/blob/main/src/main/java/com/plugish/woominecraft/WooMinecraft.java#L137

`/wp-json/wmc/v1/server/`